yt/frontends/amrvac/tests/test_read_amrvac_namelist.py

```diff
@@ -1,13 +1,19 @@
 from copy import deepcopy
 from os import path
+from pathlib import Path
 
 from yt.frontends.amrvac import read_amrvac_namelist
 from yt.testing import requires_module
 from yt.utilities.on_demand_imports import _f90nml as f90nml
 
 test_dir = path.dirname(path.abspath(__file__))
-blast_wave_parfile = path.join(test_dir, "sample_parfiles/bw_3d.par")
-modifier_parfile = path.join(test_dir, "sample_parfiles/tvdlf_scheme.par")
+blast_wave_parfile = path.join(test_dir, "sample_parfiles", "bw_3d.par")
+modifier_parfile = path.join(test_dir, "sample_parfiles", "tvdlf_scheme.par")
+
+
+@requires_module("f90nml")
+def test_read_pathlike():
+    read_amrvac_namelist(Path(blast_wave_parfile))
```

Comment:
Wouldn't this test be closer to user behavior if it wasn't passed through `Path`?

Reply:
Do you mean something like this ?
```python
    filepath = Path(blast_wave_parfile)
    read_amrvac_namelist(file path)
```

Reply:
I was meaning that I think a user is more likely to do something like `read_amrvac_namelist(blast_wave_parfile)` without using `Path` at all.

Reply:
Of course they are, but this test exist solely to check that they (... I 🙈) can do it.